NEWS.md

```diff
@@ -1,59 +1,58 @@
+    from 10. (#2961)
+ * `checkpoint`: Added support for timezone value at the end of timestamps. (#3033)
+ * `filter/rewrite`: Added `disable-jit` flag to disable JIT PCRE compilation. (#2992, #2986)
+ * `syslog-ng-ctl`: Add `export-config-graph` option to visualize config graph. (#2990)
+ * `build/dbld`: Readded CentOS 6 support. (#2860, #2971, #3028)
+ * `python`: Added Python 3.8 support. (#3017)
 
 ## Bugfixes
 
- * Fix backtick subsitution of defines/environment variables in the main configuration file. (#2906, #2909)
- * Fix SCL block parameter substitution of quoted escaped newline (#2901)
- * `python, diskq, random-generator source`: crash after failed reload (#2907)
- * Fix crash at shutdown on 32bit systems (#2893, #2895)
- * Invalidate the value of the `LEGACY_MSGHDR` macro in case either the `PID` or the `PROGRAM`
-   macros are `unset()` using a `rewrite` rule. Previously `LEGACY_MSGHDR` would retain the old values. (#2896)
- * on 32bit platform `diskq` ftruncate could fail due to size 32/64 interface (#2892)
- * Support new tzdata format, starting from version 2009.XXX, in tzinfo parser. (#2898)
- * `udp, udp6, tcp, tcp6, syslog, network destination`: Correctly detect and set `IP_MULTICAST_TTL`
-   in case of multicast ip address (#2905)
- * Fix hostname resolve on systems with only the loopback network interface configured (#2933)
- * `wildcard-file()`: Add `multi-line()`, `pad_size()` and `multi-line-mode()` option validation. (#2922)
- * `kafka-c`: Fix multiple memleaks (#2944)
+ * `tls`: Fixed an infinite loop which occured, when a `TLS` connection broke. (#3026, #3009)
+ * `log-block`: Fixed an issue, where inline `network` destinations disjointed
+   the rest of the config. (#2989, #2820)
+ * `kafka/network-load-balancer`: Fixed a crash when an argument was set to empty. (#3002)
+ * `python-source`: Fixed a memory corruption during reload. (#3014)
+ * `python-destination`: Actually use return value of `open` method. (#2998, #2513)
+ * `python-fetcher`: Fixed `FETCH_NO_DATA` and `FETCH_TRY_AGAIN` constants. (#3012)
+ * `python`: Fixed python `Exception` reporting when no `Exception` happened. (#2995)
+ * `telegram`: Fixed the syntax error of the `use-system-cert-store()` option. (#2977)
+ * `config`: Handle single dots, which were ignored before. (#3000)
+ * `file-destination`: Delay ACKs until messages are written to disk. (#2985)
+ * `http-destination`: Handle global template options values. (#3020)
+ * `timeutils`: Fixed month and day name parsing, when only the first 2 characters
+    matched. (#3035)
+ * `logmsg`: Added default `PRI` value (`LOG_USER | LOG_NOTICE`) to log messages
+   created without initial parsing. (#2974)
+ * `packaging`: Added ordering dependencies `network.target` and `network-online.target`
+   to the service files. (#2994, #2667)
+ * `amqp`: Support older (0.7.1) version (#2999)
+ * `loggen`: Set plugin path in installation time. (#3019)
```

Comment:
It can be changed at run-time with the `SYSLOGNG_PREFIX` environment variable. I'm not sure if it's important, we usually refer to this as "relocation" or "run-time relocation".